client/my-sites/plans-features-main/index.tsx

```diff
@@ -817,9 +827,11 @@ const PlansFeaturesMain = ( {
 				{ isPlansGridReady && (
 					<>
 						{ ! hidePlanSelector && (
-							<div className="plans-features-main__plan-type-selector">
```

Comment:
**Note:**

I'm changing the DOM structure to have the `StickyContainer` be the direct descendent of the appropriate HTML element in the DOM. This is because of the requirements of `sticky` styling. https://github.com/Automattic/wp-calypso/pull/85520#discussion_r1439866924

Reply:
Responded here p1704303068235729/1704249392.584319-slack-CV2TX2PAN and here p1704303465678879/1704249392.584319-slack-CV2TX2PAN

> Hmmm…not sure about the structure of the PR then. If that’s the case, then we’d probably want to forgo reusing the sticky container entirely, which forcibly assigns a sticky css attribute

> Also, I might be missing something, but how do we detect when the user has scrolled to a certain point to trigger the fixed positioning? :thinking_face: It seems like it would make our logic inherently more complex